webapps/plf-meeds-extension/src/main/webapp/WEB-INF/conf/meeds/portal/meeds/portal.xml

```diff
@@ -16,8 +16,13 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 -->
-<portal-config>
+<portal-config
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_10 http://www.gatein.org/xml/ns/gatein_objects_1_10"
+    xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_10">
     <portal-name>meeds</portal-name>
+    <displayed>true</displayed>
```

Comment:
If default value is set to `true`, no need to add it IMO

```suggestion
```

Reply:
removed